doc/external-signer.md

```diff
@@ -63,29 +65,59 @@ Under the hood this uses a [Partially Signed Bitcoin Transaction](psbt.md).
 $ bitcoin-cli -rpcwallet=<wallet> sendtoaddress <address> <amount>
 ```
 
-This prompts your hardware wallet to sign, and fail if it's not connected. If successful
-it automatically broadcasts the transaction.
+This constructs a PSBT and prompts your external signer to sign (will fail if it's not connected). If successful, Bitcoin Core finalizes and broadcasts the transaction.
 
 ```sh
-{"complete": true, "txid": <txid>}
+{"complete": true, "txid": "<txid>"}
 ```
 
 ## Signer API
 
-In order to be compatible with Bitcoin Core any signer command should conform to the specification below. This specification is subject to change. Ideally a BIP should propose a standard so that other wallets can also make use of it.
+More recent versions of Bitcoin, around v29 and later, have made significant improvements. It is recommended to use more recent versions.
```

Comment:
This documentation is shipped with Bitcoin Core. If your change is merged it will be part of the v31 release. So it's pointless to tell the user to use v29 or later.

Reply:
Yes, let's leave it out.